src/helpers/api.ts

```diff
@@ -0,0 +1,30 @@
+const API_URL = 'https://mate-academy.github.io/react_dynamic-list-of-posts/api';
+
+export const getPosts = (): Promise<Post[]> => {
+  return fetch(`${API_URL}/posts.json`)
```

Comment:
there is no need to add `.json` at the end of the url

Reply:
Without ".json" does not work. Data is not loading. These are the new links that Misha gave.